kolibri_explore_plugin/assets/src/searchApi.js

```diff
@@ -0,0 +1,56 @@
+import { KolibriApi } from './kolibriApi';
+
+function zimSearch(zimfile, query, node, maxResults = 10, start) {
```

Comment:
I would call this function `searchInsideZimFile`. Starting with the "search" verb is better to explain the function.